packages/chip/src/Chip.types.ts

```diff
@@ -0,0 +1,40 @@
+import { AnchorHTMLAttributes } from 'react';
+
+export interface ComponentProps {
+  /**
+   * Customize Chip text color
+   */
+  variant?: 'filter' | 'choice';
```

Comment:
That should be decided by the `onDelete` callback

Reply:
filter can be without onDelete too, so this is needed

<img width="555" alt="image" src="https://user-images.githubusercontent.com/106668910/187794095-b8f6e7f5-4aaf-425c-9d96-2a78a38e43a0.png">

Reply:
It needs to figure out. But generally, it is not a variant param it is a color. We should keep our main naming API convension.